doc/source/changes/version_0_26.rst.inc

```diff
@@ -94,6 +94,28 @@ Miscellaneous improvements
     >>> group_even_year
     year[2000, 2002, 2004, 2006, 2008, 2010, 2012, 2014, 2016]
 
+* allowed to perform several axes combinations at once with `combine_axes()` method (closes :issue:`382`):
+
+    >>> axes = AxisCollection('a=a0,a1;b=b0..b2;c=c0..c2;d=d0,d1')
```

Comment:
wouldn't an example using LArray.combine_axes be more telling than an example using AxisCollection? 
to avoid having a too large example I'd use: ndrange('a=a0,a1;b=b0,b1;c=c0,c1;d=d0..d2')

Reply:
regardless of the example, you should at least mention that it is implemented both for LArrray and AxisCollection